zebra/rib.h

```diff
@@ -323,7 +323,8 @@ enum rib_update_event {
 
 extern void route_entry_copy_nexthops(struct route_entry *re,
 				      struct nexthop *nh);
-int route_entry_update_nhe(struct route_entry *re, struct nhg_hash_entry *new);
+int route_entry_update_nhe(struct route_entry *re,
+			   struct nhg_hash_entry *new_re);
```

Comment:
how about new_nhghe, new_re will showup in searches for `struct route_entry` and that makes me a sad panda

Reply:
I should have probably spent a second reading the actual struct type before renaming it... I'll fix this